manual/guides/courses.md

```diff
@@ -1,27 +1,108 @@
 ---
 layout: manual
 title: Course Planning & Registration
-subtitle: "Information on Choosing and Registering for Classes"
+subtitle: "Assistance with Choosing and Registering for Classes"
```

Comment:
```suggestion
subtitle: "Choosing and Registering for Classes"
```

Reply:
Hi Gavin -- it looks like you've resolved this without committing the suggestion.... Let's discuss how to handle these change suggestions.